File: SampleDemo/app/src/main/java/com/adform/adformdemo/SampleActivity.java
package com.adform.adformdemo;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.adform.sdk.utils.AdSize;

/**
 * Created by mariusmerkevicius on 5/4/15.
 */
public class SampleActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_sample);
    }

    public void openAdInline(View v) {
        startActivity(new Intent(this, AdInlineActivity.class));
    }

    public void openAdInlineCustom1(View v) {
        Intent intent = new Intent(this, AdInlineActivityBundle.class);
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_MASTER_TAG, 4672767);
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_AD_SIZE, new AdSize(300, 250));
        startActivity(intent);
    }

    public void openAdInlineCustom2(View v) {
        Intent intent = new Intent(this, AdInlineActivityBundle.class);
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_MASTER_TAG, 4022668);
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_AD_SIZE, new AdSize(320, 50));
        startActivity(intent);
    }

    public void openAdInlineCustom3(View v) {
        Intent intent = new Intent(this, AdInlineActivityBundle.class);
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_MASTER_TAG, 4022833);
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_AD_SIZE, new AdSize(320, 50));
        startActivity(intent);
    }

    public void openAdInlineCustom4(View v) {
        Intent intent = new Intent(this, AdInlineActivityBundle.class);
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_MASTER_TAG, 4015369);
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_AD_SIZE, new AdSize(320, 50));
        startActivity(intent);
    }

    public void openAdOverlayCustom1(View v) {
        Intent intent = new Intent(this, AdOverlayActivityBundle.class);
        intent.putExtra(AdOverlayActivityBundle.BUNDLE_KEY_MASTER_TAG, 4660166);
        startActivity(intent);
    }

    public void openAdOverlayCustom2(View v) {
        Intent intent = new Intent(this, AdOverlayActivityBundle.class);
        intent.putExtra(AdOverlayActivityBundle.BUNDLE_KEY_MASTER_TAG, 4935199);
        startActivity(intent);
    }

    public void openAdInlineLW(View v) {
        startActivity(new Intent(this, AdInlineLWActivity.class));
    }

    public void openAdOverlay(View v) {
        startActivity(new Intent(this, AdOverlayActivity.class));
    }

    public void openAdInterstitial(View v) {
        startActivity(new Intent(this, AdInterstitialActivity.class));
    }

    public void openAdHesion(View v) {
        startActivity(new Intent(this, AdHesionActivity.class));
    }

}

File: SampleDemo/app/src/main/java/com/adform/adformdemo/SampleActivity.kt
package com.adform.adformdemo

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.WindowCompat
import com.adform.sdk.utils.AdSize

class SampleActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        WindowCompat.setDecorFitsSystemWindows(window, false)
        setContentView(R.layout.activity_sample)
    }

    fun openAdInline(v: View?) {
        startActivity(Intent(this, AdInlineActivity::class.java))
    }

    fun openAdInlineCustom1(v: View?) {
        val intent = Intent(this, AdInlineActivityBundle::class.java)
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_MASTER_TAG, 4672767)
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_AD_SIZE, AdSize(300, 250))
        startActivity(intent)
    }

    fun openAdInlineCustom2(v: View?) {
        val intent = Intent(this, AdInlineActivityBundle::class.java)
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_MASTER_TAG, 4022668)
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_AD_SIZE, AdSize(320, 50))
        startActivity(intent)
    }

    fun openAdInlineCustom3(v: View?) {
        val intent = Intent(this, AdInlineActivityBundle::class.java)
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_MASTER_TAG, 4022833)
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_AD_SIZE, AdSize(320, 50))
        startActivity(intent)
    }

    fun openAdInlineCustom4(v: View?) {
        val intent = Intent(this, AdInlineActivityBundle::class.java)
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_MASTER_TAG, 4015369)
        intent.putExtra(AdInlineActivityBundle.BUNDLE_KEY_AD_SIZE, AdSize(320, 50))
        startActivity(intent)
    }

    fun openAdOverlayCustom1(v: View?) {
        val intent = Intent(this, AdOverlayActivityBundle::class.java)
        intent.putExtra(AdOverlayActivityBundle.BUNDLE_KEY_MASTER_TAG, 4660166)
        startActivity(intent)
    }

    fun openAdOverlayCustom2(v: View?) {
        val intent = Intent(this, AdOverlayActivityBundle::class.java)
        intent.putExtra(AdOverlayActivityBundle.BUNDLE_KEY_MASTER_TAG, 4935199)
        startActivity(intent)
    }

    fun openInStreamActivity(v: View?) {
        val intent = Intent(this, InStreamActivity::class.java)
        intent.putExtra(InStreamActivity.BUNDLE_KEY_MASTER_TAG, 1505418)
        startActivity(intent)
    }

    fun openAdInlineLW(v: View?) {
        startActivity(Intent(this, AdInlineLWActivity::class.java))
    }

    fun openAdOverlay(v: View?) {
        startActivity(Intent(this, AdOverlayActivity::class.java))
    }

    fun openAdInterstitial(v: View?) {
        startActivity(Intent(this, AdInterstitialActivity::class.java))
    }

    fun openAdWebInterstitial(v: View?) {
        startActivity(Intent(this, AdInterstitialWebActivity::class.java))
    }

    fun openAdHesion(v: View?) {
        startActivity(Intent(this, AdHesionActivity::class.java))
    }

    fun openAdNative(v: View?) {
        startActivity(Intent(this, AdNativeActivity::class.java))
    }
}
